packages/react-native/example/tsconfig.json

```diff
@@ -1,20 +1,10 @@
+// prettier-ignore
 {
-    "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
-        "jsx": "react-native",
-        "noEmit": true,
-        "strict": true,
-        "moduleResolution": "node",
-        "baseUrl": "./", /* Base directory to resolve non-absolute module names. */
-        "esModuleInterop": true
```

Comment:
not sure which of these we'll still need vs the new `@tsconfig/react-native/tsconfig.json` that shows up for extension

Reply:
i've changed the template so it will already have ts setup correctly out of the box 🙏